packages/snaps-execution-environments/.c8rc.json

```diff
@@ -4,8 +4,9 @@
     "*.js",
     "**/index.ts",
     "**/*.ava.test.ts",
-    "**/executor.ts",
-    "**/endowments.ts"
+    "**/test-utils/executor.ts",
+    "**/test-utils/endowments.ts",
+    "**/test-utils/hardening.ts"
```

Comment:
```suggestion
    "**/test-utils/**"
```

Reply:
Thanks for the suggestion! This actually works, I tried previously pattern similar to the one I added in Jest config, but it was not working.